migrations/20240806054557_template_category.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS af_template_category (
+  category_id   UUID NOT NULL DEFAULT gen_random_uuid(),
+  created_at    TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at    TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
maybe your intention is to use a trigger to update the `updated_at` field when there's change in the row. i dont see this field in the sqlx sql. if you dont need to the updated time, can also remove this column altogether

Reply:
It was my mistake, i didn't include it on the sqlx sql. As of the moment this field is not being used for frontend, though i do think there is a good possibility that that may change in the future.